website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -435,12 +441,19 @@ memcached-sample                        5/5     5            5           3m
 
 ### Cleanup
 
+A new target can be added into the Makefile for cleaning up the resources that have been created along this tutorial:
+
 ```sh
-$ kubectl delete -f config/samples/cache_v1alpha1_memcached.yaml
-$ kubectl delete deployments,service -l control-plane=controller-manager
-$ kubectl delete role,rolebinding --all
+# UnDeploy controller from the configured Kubernetes cluster in ~/.kube/config
```

Comment:
```suggestion
# Undeploy controller from the configured Kubernetes cluster in ~/.kube/config
```
is there is a reason for the capitol "D"?

Reply:
This comes from: https://github.com/operator-framework/operator-sdk/issues/4146#issuecomment-719499818
I am happy to use the right spelling if it is consistent with what will be generated in the future

Reply:
+1 to use `Undeploy`. I agree that is better.